lib/client.ts

```diff
@@ -165,6 +165,20 @@ export default class Client {
       .then(res => res.richmenus);
   }
 
+  public setDefaultRichMenu(richMenuId: string): Promise<any> {
+    return this.post(URL.defaultRichMenu(richMenuId));
+  }
+
+  public getDefaultRichMenuId(): Promise<string> {
+    return this.get(URL.defaultRichMenu())
+      .then(checkJSON)
+      .then(res => res.richMenuId);
+  }
+
+  public deleteDefaultRichMenu(): Promise<any> {
```

Comment:
Return type may be `Promise<{}>` or `Promise<void>`. Please choose what you prefer.